pizzapi/order.py

```diff
@@ -35,14 +41,25 @@ def remove_item(self, code):
         codes = [x['Code'] for x in self.data['Products']]
         return self.data['Products'].pop(codes.index(code))
 
+    def add_coupon(self, code, qty=1):
+        item = self.menu.variants[code]
+        item.update(ID=1, isNew=True, Qty=qty, AutoRemove=False)
+        self.data['Coupons'].append(item)
+        return item
+
+    def remove_coupon(self, code):
+        codes = [x['Code'] for x in self.data['Coupons']]
+        return self.data['Coupons'].pop(codes.index(code))
+
     def _send(self, url, merge):
         self.data.update(
             StoreID=self.store.id,
             Email=self.customer.email,
             FirstName=self.customer.first_name,
             LastName=self.customer.last_name,
             Phone=self.customer.phone,
-            Address=self.customer.address.data
+            #Address=self.address.street
+
```

Comment:
Why comment this out?

Reply:
that line is no longer needed because of the payload change in 'class Order'.  it was originally being set in def __send but was only updating it as a single string which was causing a "BusinessRule ServiceMethodAllowedAtStore failed. Cause: nullOnStoreID" error.

to make things easier when i initialize Order i just pass 'address' along with it and set the dictionary key values to the individual parts of the address.  see the Quick Start on the Readme where i do 'address = Address(*customer.address.split(','))' to break it up as a list instead of a single string.  it kind of kills two birds with one stone because of the str not callable error